# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mesa/fdo-5835.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

See https://bugs.freedesktop.org/show_bug.cgi?id=5835
	--jsaw

--- Mesa/src/mesa/main/glheader.h	2006/02/11 02:59:38	1.70
+++ Mesa/src/mesa/main/glheader.h	2006/02/15 21:03:34	1.71
@@ -46,6 +46,9 @@
 #ifndef GLHEADER_H
 #define GLHEADER_H

+#if defined (HAVE_DIX_CONFIG_H)
+# include <dix-config.h>
+#endif
 
 #if defined(XFree86LOADER) && defined(IN_MODULE) && !defined(NO_LIBCWRAPPER)
 #include "xf86_ansic.h"

--- Mesa-6.4.2/src/mesa/drivers/dri/common/glcontextmodes.c	2005-11-30 00:02:16.000000000 +0100
+++ Mesa-6.4.2-t2/src/mesa/drivers/dri/common/glcontextmodes.c	2006-02-27 11:13:07.000000000 +0100
@@ -39,6 +39,9 @@
 # include "imports.h"
 # define __glXMemset  memset
 #else
+# if defined (HAVE_DIX_CONFIG_H)
+#  include <dix-config.h>
+# endif
 # include <X11/X.h>
 # include <GL/glx.h>
 # include "GL/glxint.h"
